recipes/pagmo2/pre_2.11/patches/0002-fix-cmake-variables-2-10.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 6d310a2c..012cb264 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -116,13 +116,21 @@ if(PAGMO_BUILD_PAGMO)
+     # Eigen3
+     if(PAGMO_WITH_EIGEN3)
+         find_package(Eigen3 REQUIRED)
++        if(Eigen3_FOUND)
++            set(EIGEN3_INCLUDE_DIR ${Eigen3_INCLUDE_DIR})
++            set(EIGEN3_VERSION ${Eigen3_VERSION})
++        endif()
```

Comment:
what is the purpose of these variables if you link eigen target afterwards?

Reply:
In this case only for printing the correct path when creating the package, should I remove this again?